Nitrox.Server.Subnautica/Server.cs

```diff
@@ -281,22 +281,30 @@ private async Task LogHowToConnectAsync()
         Task<IPAddress> wanIp = NetHelper.GetWanIpAsync();
         Task<IEnumerable<(IPAddress Address, string NetworkName)>> vpnIps = Task.Run(NetHelper.GetVpnIps);
 
-        List<string> options = ["127.0.0.1 - You (Local)"];
-        if (await wanIp != null)
+        List<string> options = [$"{IPAddress.Loopback} - You (Local)"];
+        IPAddress? wanAddress = await wanIp;
+        if (wanAddress != null)
         {
             options.Add("{ip:l} - Friends on another internet network (Port Forwarding)");
         }
         foreach ((IPAddress? vpnAddress, string? vpnName) in await vpnIps)
         {
-            options.Add($"{vpnAddress} - Friends using {vpnName} (VPN)");
+            if (vpnAddress == null)
+            {
+                continue;
+            }
+            IPAddress displayAddress = vpnAddress.TryGetAsIPv4();
```

Comment:
Happy with this for now but do we want to always have it as ipv4? Say someone is expecting it to be ipv6 as they have entered that intentionally we would not display that here